### PR TITLE
added a blog

### PR DESCRIPTION
**Pull request by zcr20090430**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.  
    For example:  

    ``` md
    ---
    title: "Your Great Blog Title"
    author: your_github_id
    image: /assets/2020/08-your-blog-slug/teaser.jpg
    ---

    YOUR_FIRST_SENTENCE_SHOULD_BE_A_SUMMARY_FOR_YOUR_POST
    ```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2020` directory.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.js.org/blob/master/_developers/YOUR_GITHUB_USERNAME.md> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!
  For excample:
  ```md
  ---
  name: Jiarui LI (李佳芮)
  site: https://lijiarui.github.io
  bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
  avatar: /assets/developers/lijiarui/avatar.png
  email: rui@juzi.bot
  twitter: https://weibo.com/u/2175505900
  ---
  ```
  Here is the example file:
  - lijiarui.md: https://github.com/wechaty/wechaty.github.io/blob/master/_developers/lijiarui.md
  - It will shown like this on wechaty blog: https://wechaty.github.io/developers/lijiarui/

To read more about the contribute guideline, visit: <https://github.com/wechaty/wechaty.js.org#contribute-guideline>

### CODE REVIEW

**Comment by zcr20090430** [comment]
我需要做什么？

File: _developers/zcr19960116.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: padlocal
+site: https://github.com/padlocal
+avatar: /assets/developers/padlocal/avatar.png
```
Comment (huan) [inline _developers/zcr19960116.md]: Please upload your avatar.png in the PR as well

**Review by huan — CHANGES_REQUESTED** [review]
Are you 好大?

Please move the `_developers/` and `_posts` directory to `jekyll/` because the latest master branch directory structure has been changed.

After you apply those changes, this PR will be ready to be mreged.

**Review by huan — APPROVED** [review]
LGTM

**Review by wj-Mcat — APPROVED** [review]
LGTM

**Comment by huan** [comment]
@zcr20090430  Thank you very much for your sharing!

> You've invited orangeray to Wechaty! They'll be receiving an email shortly. They can also visit https://github.com/wechaty to accept the invitation.

I have invited you to join our Wechaty GitHub Organization, please accept it by following the above message.

And we also have a WeChat room for contributors only which can discuss Wechaty at a deep level, you are welcome to join and if you are interested please contact @lijiarui to ask for an invitation.

**Pull request merged**